monero-wallet-rpc/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04 AS install
+
+ARG VRS
+ENV VERSION=${VRS:-v0.17.2.3}
+RUN echo $VERSION
+
+RUN apt update && apt install -y wget
+# Get bins and signatures
+RUN wget https://downloads.getmonero.org/cli/monero-linux-x64-$VERSION.tar.bz2
+RUN echo "8069012ad5e7b35f79e35e6ca71c2424efc54b61f6f93238b182981ba83f2311 monero-linux-x64-$VERSION.tar.bz2" | sha256sum -c
+
+# Extract and install binaries
+RUN tar xf monero-linux-x64-$VERSION.tar.bz2
+RUN install -m 0755 -o root -g root -t /usr/bin monero-x86_64-linux-gnu-$VERSION/monero*
+
+FROM ubuntu:20.04
+
+COPY --from=install /usr/bin/monero* /usr/bin/
+
+# monero-wallet-rpc port
+EXPOSE 18083
+
+VOLUME /wallets
```

Comment:
_(no _big_ deal, just an opinion):_

I'm particularly not a fan of `VOLUME ..` declarations in Dockerfiles as it's easy to end up with anonymous volumes by doing something like `docker run <image> ..` without specifying `-v` and then, not knowing that the Dockerfile declared a volume, end up _thinking_ that I only had ephemeral data but no, there's still stuff on disk even after the container got deleted (sure, a `docker volume prune` or similar would get rid of it, but IMO, best practice is to not have that in Dockerfile).

was there a particular reason to make use of it?

Reply:
Agree that volumes might be sometimes hard to manage, it is very easy to mess up. But we need shared access between containers, for `bitcoin-core` we need the `/data` for access to the `.cookie` file and use `electrs`, here I guess @TheCharlatan needs `/wallets` later if we want to use other software on-top of it or need direct access to the files.

Reply:
I'll remove it actually. I thought I would not need it to read some stuff locally, but I did not end up needing. Thanks for the perspective @cirocosta .

Reply:
I just pushed some changes, but I'm done editing. I let you remove what you want.

Reply:
Ok, done with my changes. Thanks for doing all the publishing :)